### DISCUSSION TITLE
How to get all value from a multiple select in Remix ActionFunction?

### DISCUSSION THREAD

**Discussion by mrmarufpro** [remix-run/remix#4711] [Q&A] [answered]
I named the `select` tag with `categories` and try to get the value in the ActionFuntion by using `formData()`. But the result is always one value from the `select` tag. How can i get all the selected value? 
Thanks in advance.

![image](https://user-images.githubusercontent.com/39343312/204719354-af00ff4e-3ee0-46a3-838e-40b137a8f68d.png)
![image](https://user-images.githubusercontent.com/39343312/204719370-a1ccb824-2a6d-48e1-a5e1-1dd8d917719a.png)
![image](https://user-images.githubusercontent.com/39343312/204719381-8b01c3a4-bd6d-4aa3-aa4d-4794f20f9939.png)
![image](https://user-images.githubusercontent.com/39343312/204719391-22d4189d-f420-4ae4-bf96-f60e67c8a4cf.png)

**Comment by machour** [comment, marked as answer]
Use `form.getAll()`, see https://developer.mozilla.org/en-US/docs/Web/API/FormData/getAll

**Reply by mrmarufpro** [reply]
Thank you. It's solved my problem.
